data/hai/unfettered side missions.txt

```diff
@@ -65,3 +65,471 @@ mission "Unfettered returning home"
+			`	"Of course, the ship is aged, and you should not expect the best price in a shipyard. Nevertheless, it is a fine vessel, capable of fighting, freighting, and mining. So, what do you say, <last>? This is the opportunity of a lifetime; they simply don't make ships like this anymore."`
+			label choice
+			choice
+				`	"I'll take the... Anomalocaris, was it?"`
+					goto take
+				`	"If you lack crew, who was piloting the ship from the ocean?"`
+					to display
+						not "unfettered: pilot dialog"
+					goto pilot
+				`	"Why was the other ship submerged?"`
+					to display
+						not "unfettered: ocean dialog"
+					goto ocean
+				`	"No, thanks. I'm not looking to add another ship to my fleet."`
+			`	The two Hai look disappointed. "A shame. We will probably not meet again. Farewell," one says. They signal the ship from earlier to submerge into the ocean again and walk off along the beach.`
+				goto "train back"
+			label pilot
+			action
+				set "unfettered: pilot dialog"
+			`	"Actually, it's a 'what,' not a-" starts the first one, but they are again interrupted by the second.`
+			`	"The less you know, the better for all of us, Captain."`
+				goto choice
+			label ocean
+			action
+				set "unfettered: ocean dialog"
+			`	"We found it there," the first one answers.`
+			`	"It is hard to find a drop of rain in a storm, but the drop of rain may see all," the other one adds.`
+				goto choice
+			label take
+			action
+				give ship "Anomalocaris (Venerable)" "Good Omen"
+					id "hai legacy"
+				fine 10000000
+			`	They give you access to the ship and proceed to explain the details. "It is of no real significance, but surely you understand we cannot go through standard means for this sale. We will instead present it as a fine. You'll be charged for carrying technology you had no idea was forbidden, and it will not go on your permanent record. These Korath weapons are suspicious-looking indeed."`
+			`	You quickly realize that they are not asking you for your opinion at all. In fact, you receive notice of the fine before they have even finished speaking. "We will need to make some modifications to the ship before you can use it.`
+			`	"Oh, I almost forgot: make sure you return to the spaceport to get the cargo loaded onto your ship. You should use this ship and this ship alone to make these urgent deliveries. Arriving in a different ship or with a supporting fleet will unnerve the Unfettered."`
+			choice
+				`	(Go back to the shipyard.)`
```

Comment:
The above says that using another ship will upset the Unfettered, but does not elaborate. Maybe have another choice here to explain why.